test/test_stories.md

```diff
@@ -0,0 +1,12 @@
+## feedback2
+    - utter_ask_feedback
+* enter_data
+    - utter_thumbsup
+    - utter_anything_else
+
+## akela test
+* enter_data@0.7{"name":"Akela"}
+    - action_store_entity_extractor
+    - slot{"entity_extractor": "SpacyEntityExtractor"}
+    - utter_spacy
+    - utter_anything_else
```

Comment:
I like the idea, I think we should only test on flows we've verified. The second one here doesn't look correct, the first will ignore the utter_ask_feedback i think, which makes the repsonse after enter_data unpredictable (again, i think)

Reply:
With the second one, I was trying to test #351 that Akela reported.

The first example comes directly from `feedback.md` and the entire story file looks incorrect.

Reply:
Replaced with single working story in latest commit.